package.json

```diff
@@ -51,10 +51,15 @@
     "coffee-errors": "^0.8.6",
     "coffee-script": "~1.10.0",
     "csonschema": "^0.5.1",
+    "fs": "0.0.2",
     "glob": "^7.0.3",
+    "jsonlint": "^1.6.2",
     "mocha": "~2.5.1",
+    "node-rest-client": "^1.5.1",
+    "optimist": "^0.6.1",
     "mustache": "~2.2.1",
     "proxyquire": "^1.3.1",
+    "q": "^1.4.1",
```

Comment:
Jesus, you also removed `node-rest-client` which was there from one of our last bug fix releases for using Grunt. Please put **ALL** of them back since no one checked what was actually needed first! `optimist` was replaced by `yargs`, but it's been gone for a long time.